frontend/package.json

```diff
@@ -37,6 +37,8 @@
     "test-cypress-devconsole-headless": "cd packages/dev-console/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/project-creation.feature\";",
     "test-cypress-ceph": "cd packages/ceph-storage-plugin/integration-tests-cypress && ../../../node_modules/.bin/cypress open --config-file cypress-ceph.json --env openshift=true",
     "test-cypress-ceph-headless": "cd packages/ceph-storage-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --config-file cypress-ceph.json --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-kubevirt": "cd packages/kubevirt-plugin/integration-tests-cypress && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-kubevirt-headless": "cd packages/kubevirt-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
```

Comment:
@dtaylor113 We should investigate a way to add these tests that avoids two need commands in the package.json file every time. Maybe a script that takes a parameter or env var.

Nothing to change in this PR, just an observation.

Reply:
Hi @spadgett, created https://issues.redhat.com/browse/CONSOLE-2538 to track